tasks/tool.py

```diff
@@ -111,6 +112,33 @@ def get_image_description(ctx: Context, ami_id: str) -> Any:
         return result["Images"][0]
 
 
+def notify(text):
+    if is_linux():
+        notify_linux()
+    if is_windows():
+        notify_windows()
+    notify_macos(text)
+
+
+def notify_macos(text):
+    CMD = '''
+    on run argv
+    display notification (item 2 of argv) with title (item 1 of argv)
+    end run
+    '''
+    subprocess.call(['osascript', '-e', CMD, "test-infra-definitions", text])
+
+
+def notify_linux():
+    # TODO: Implement notification on linux. Would require linux computer (with desktop) to test
+    return
+
+
+def notify_windows():
+    # TODO: Implenent notification on windows. Would require windows computer (with desktop) to test
```

Comment:
As a follow up, should use [win10toast](https://stackoverflow.com/questions/12575708/notification-using-python)

Reply:
Can I create follow up cards for this? I saw that some solution exist but since I will not be able to test it I think it is not super safe to try to implement it. It could broke windows/linux users.
Or I can wrap all this code in a `try` and do not fail even if my code do not work

Reply:
Yes absolutely ! You can add cards mentioning this, if a linux or windows dev needs it they will have a trace to it